suite/tests/CMakeLists.txt

```diff
@@ -2333,11 +2333,15 @@ if (CLIENT_INTERFACE)
   endif ()
 
   if (X86) # Generated code is x86-specific.
+    set(checklevel "")
+    if (DEBUG)
+      set(checklevel "-checklevel 0")
+    endif ()
     tobuild_api(api.ibl-stress api/ibl-stress.c
       # This tests indirect branches between blocks.
       # We use -checklevel 0 to disable the DOCHECK in check_thread_vm_area
       # which makes building 150K bbs very slow.
```

Comment:
Seems like the comment should go up by the actual instance of -checklevel 0

Reply:
Done